### ISSUE TITLE
SQL dump for Participant Datastore failing

### ISSUE THREAD

**Issue by moschetti** [GoogleCloudPlatform/fda-mystudies#2534]
**Describe the bug**
Participant Datastore SQL dump failing with 2 errors

**To Reproduce**
1. Copy file to GCS:
`gsutil cp ./participant-datastore/sqlscript/mystudies_participant_datastore_db_script.sql gs://<prefix>-<env>-mystudies-sql-import`
3. Try to import
`gcloud sql import sql --project=<prefix>-<env>-data <instance-name> gs://<prefix>-<env>-mystudies-sql-import/mystudies_participant_datastore_db_script.sql`

4. Fails with:
  ```
  ERROR: (gcloud.sql.import.sql) [ERROR_RDBMS] exit status 1

  ERROR 1273 (HY000) at line 435: Unknown collation: 'utf8mb4_0900_ai_ci'
  ```
5. This error can be fixed by making the change in #2517 and re-copying the file
6. Retry steps above, and fails with the following error
`ERROR 1215 (HY000) at line 435: Cannot add foreign key constraint`

**Additional context**
Issue is occurring when deploying from a clone of master branch. I also checked develop branch, but the file is the same there, causing the same errors.

**Comment by moschetti** [comment]
Looks like the issue is in lines 434-459 which was changed a few times in the past week, so the break likely came in one of those commits: 
https://github.com/GoogleCloudPlatform/fda-mystudies/blame/master/participant-datastore/sqlscript/mystudies_participant_datastore_db_script.sql

**Comment by aswinijena100** [comment]
@moschetti please let know , if we can close this ticket

**Comment by aswinijena100** [comment]
Closing this ticket as this issue is fixed with PR #2541
